.github/workflows/ci_cd.yml

```diff
@@ -56,10 +62,10 @@ jobs:
       - name: Get image tag
         id: get-image-tag
         run: |
-          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
-              echo "image_tag=${{ inputs.image_tag }}" >> "$GITHUB_OUTPUT"
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" && "${{ inputs.image_tag }}" != "" ]]; then
+              echo "image_tag=${{ inputs.image_tag }}" | tee "$GITHUB_OUTPUT"
```

Comment:
Was this change so that we could also see it in the workflow run?

Reply:
Yes, since it's not a secret, it's good to have some visibility into the value.